knowledge-base/implementation-documentation/0005-api-improvements.md

```diff
@@ -214,6 +214,54 @@ Cons:
 - Difficult to implement.
 - Not realistic for 2.0, but maybe later.
 
+## Function style filtering [Updated]
+
+The new filter api is implemented as non-breaking change and the current methods are deprecated. Lambda filters on collection properties will be implemented as a new feature in the major version.
```

Comment:
[q] What is the difference between a `colleciton property` and  the `TO_FRIENDS` one to many relation?

Reply:
Generally, a collection property is a list of either basic or complex type and 1-N is a list of another entity type. 
In terms of filters, there are represented as [CollectionField](https://github.com/SAP/cloud-sdk-js/blob/main/packages/odata-common/src/selectable/collection-field.ts#L17) and [OneToManyLink](https://github.com/SAP/cloud-sdk-js/blob/main/packages/odata-common/src/selectable/one-to-many-link.ts) respectively.